src/mam4xx/mo_gas_phase_chemdr.hpp

```diff
@@ -430,9 +431,10 @@ void perform_atmospheric_chemistry_and_microphysics(
     if (0 <= ll) {
       const auto aqso4 = ekat::subview(dqdt_aqso4, ll);
       const auto aqh2so4 = ekat::subview(dqdt_aqh2so4, ll);
-      const Real vmr_so4 = aero_model::calc_sfc_flux(team, aqso4, pdel, nlev);
-      const Real vmr_h2s = aero_model::calc_sfc_flux(team, aqh2so4, pdel, nlev);
-      aqso4_flx[m] =
+      //FIXME: The calc_sfc_flux routine or another routine is producing a 'Bad file descriptor' error in Frontier.
+      const Real vmr_so4 = 0.0;//aero_model::calc_sfc_flux(team, aqso4, pdel, nlev);
+      const Real vmr_h2s = 0.0;//aero_model::calc_sfc_flux(team, aqh2so4, pdel, nlev);
+      aqso4_flx[m] = 
```

Comment:
We can remove these lines along with the following two lines:
```
      aqso4_flx[m] =      
          conversions::mmr_from_vmr(vmr_so4, adv_mass_kg_per_moles[ll]);
      aqh2so4_flx[m] =
          conversions::mmr_from_vmr(vmr_h2s, adv_mass_kg_per_moles[ll]);
```
These are extra diags, not asked by @kaizhangpnl. We can enable them later once we figure out the real issue.